### PR TITLE
add: `grub-customizer-git`

### PR DESCRIPTION
**Pull request by ObsidianPresidium**

### CODE REVIEW

File: packages/grub-customizer-git/grub-customizer-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name="grub-customizer-git"
+pkgname="grub-customizer"
+url="https://git.launchpad.net/${pkgname}"
+build_depends="cmake g++ libgtkmm-3.0-dev gettext libssl-dev libarchive-dev hwinfo"
+breaks="${pkgname}-deb"
+description="Grub Customizer is a graphical interface to configure the GRUB2/BURG settings and menuentries"
+maintainer="Obsidian <emil@aarestrup.nu>"
+repology=("project: ${pkgname}")
```
Comment (wizard-28) [inline packages/grub-customizer-git/grub-customizer-git.pacscript:8]:
```suggestion
```

Repology is invalid in `-git` pacscripts.

File: packages/grub-customizer-git/grub-customizer-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,31 @@
+name="grub-customizer-git"
+pkgname="grub-customizer"
+url="https://git.launchpad.net/${pkgname}"
+build_depends="cmake g++ libgtkmm-3.0-dev gettext libssl-dev libarchive-dev hwinfo"
+breaks="${pkgname}-deb"
+description="Grub Customizer is a graphical interface to configure the GRUB2/BURG settings and menuentries"
+maintainer="Obsidian <emil@aarestrup.nu>"
```
Comment (wizard-28) [inline packages/grub-customizer-git/grub-customizer-git.pacscript:7]:
```suggestion
maintainer="ObsidianPresidium <emil@aarestrup.nu>"
```

Consider using your GitHub username for automatic issue assignments.

File: packages/grub-customizer-git/grub-customizer-git.pacscript [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+gives="${pkgname}"
+
+pkgver() {
+  git ls-remote "${url}" refs/heads/master | cut -f1 | cut -c1-8
+}
+version="${pkgver}"
+
+build() {
+  cmake .
+  make
```
Comment (wizard-28) [inline packages/grub-customizer-git/grub-customizer-git.pacscript:18]:
```suggestion
  make -j"$(nproc)"
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/grub-customizer-git/grub-customizer-git.pacscript (hunk 2 above)
Comment (ObsidianPresidium) [reply packages/grub-customizer-git/grub-customizer-git.pacscript:7]: I use the name Obsidian in all my other pacscripts, I'll have to update those as well sometime I assume?

Comment (wizard-28) [reply packages/grub-customizer-git/grub-customizer-git.pacscript:7]: Yeah

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Edit `packagelist`

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**